packages/SketchMorph2-Core.package/M2DrawingContext.class/instance/currentTool.st

```diff
@@ -0,0 +1,3 @@
+accessing
+currentTool
+	^ currentTool ifNil: [currentTool := #pen]
```

Comment:
Shouldn't this be tied to the setting which button is the default?

Reply:
We decided to ignore the coupling to the default button since this was not the case before this PR and is out of scope of this PR and also no trivial change.